extensions/cloud/portability-cloud-local/src/main/java/org/dataportabilityproject/cloud/local/LocalJobStore.java

```diff
@@ -16,21 +16,30 @@
 package org.dataportabilityproject.cloud.local;
 
 import com.google.common.base.Preconditions;
+import org.dataportabilityproject.spi.cloud.storage.JobStore;
+import org.dataportabilityproject.spi.cloud.types.JobAuthorization;
+import org.dataportabilityproject.spi.cloud.types.PortabilityJob;
+
 import java.io.IOException;
 import java.util.Map;
 import java.util.Map.Entry;
 import java.util.UUID;
 import java.util.concurrent.ConcurrentHashMap;
-import org.dataportabilityproject.spi.cloud.storage.JobStore;
-import org.dataportabilityproject.spi.cloud.types.JobAuthorization;
-import org.dataportabilityproject.spi.cloud.types.PortabilityJob;
 
 /** An in-memory {@link JobStore} implementation that uses a concurrent map as its store. */
 public final class LocalJobStore implements JobStore {
-  private final ConcurrentHashMap<UUID, Map<String, Object>> map;
+  private static ConcurrentHashMap<UUID, Map<String, Object>> SINGLETON_MAP;
```

Comment:
As discussed on the call, I think you can assume that if this store is being used, we want the singleton

The other local impl uses GoogleJobStore backed by DataStore local emulator

I think we can get rid of the singleVM flag, and instead have LocalCloudExtension use the 'cloud' and 'env' flags to determine whether to return LocalJobStore or one of the emulator stores (I can do that along with #234)

Reply:
I removed the singleVM flag. If you can re-work it to use the other flags in 234 that would be great.